README.md

```diff
@@ -1,2 +1,70 @@
-# cmo-metadb-label-generator
-CMO MetaDB messaging service for handling CMO label generation
+# CMO Label Generator 🔍
+
+CMO Label Generator is a distributed microservices system. It receives messages from LIMS when a request is marked delivered. This message is processed and a unique CMO label Id is generated. Any new message is then published to downstream subscribers.
```

Comment:
I think it should be clearer that this generator is providing Sample labels (aka ids) following a technical specification.  I would add the PDF to the repos and link to it.

We should mention that the interface to this is the publishing of a message on the topic specified in application properties.   We should state that the message content is a JSON representation of SampleManifest (can we point to a definition someplace?).

We should mention that there is a dependency on MetaDB and that a RequestReply mechanism (also set in the property file) is used to determine the patient sample set.